### PR TITLE
Non-Modular I was forced to state laws by a jaunting miner

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Removes bloodcrawl from bubblegums loot pool

## Why It's Good For The Game
Miners shouldn't have a jaunt that lets them be invulnerable forever and go wherever they please before manifesting somewhere with blood

## Proof Of Testing
I loaded the game up and killed bubblegum a few times, only got the soul scythe

## Changelog

:cl:
Deletes bloodcrawl from bubblegums loot pool
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -124,12 +124,7 @@
 
 /obj/structure/closet/crate/necropolis/bubblegum/PopulateContents()
 	new /obj/item/clothing/suit/hooded/hostile_environment(src)
```
Comment (Erol509) [inline code/modules/mining/lavaland/necropolis_chests.dm:126]: Dont remove it, comment it out. Use /* */ block comment and name it Bubberstation removal start at the begining and END at the end

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -124,12 +124,7 @@
...
-	var/loot = rand(1,2)
```
Comment (Erol509) [inline code/modules/mining/lavaland/necropolis_chests.dm:127]: Dont remove it, comment it out. Use /* */ block comment and name it Bubberstation removal start at the begining and at the end

**Review by Erol509 — CHANGES_REQUESTED** [review]

**Comment by ReturnToZender** [comment]
Won the court case

After the PR is merged

**Comment by MLGTASTICa** [comment]
Could put a little bit more effort and make it be a shittier blood-jaunt which is interrupted when the blood poll they are sitting in is cleaned and that cant teleport to other blood polls.

**Comment by Odairu** [comment]
> Could put a little bit more effort and make it be a shittier blood-jaunt which is interrupted when the blood poll they are sitting in is cleaned and that cant teleport to other blood polls.

No, the spell is toxic and miners shouldn't be the literal gods of the station. This was originally a skyrat addition because they don't know what balance in a video game is. Miners shouldn't get to play dota 2 and come back on station an hour into the game and be able to do whatever the fuck they want to whoever they want with zero recourse.

**Comment by MLGTASTICa** [comment]
> > Could put a little bit more effort and make it be a shittier blood-jaunt which is interrupted when the blood poll they are sitting in is cleaned and that cant teleport to other blood polls.
> 
> No, the spell is toxic and miners shouldn't be the literal gods of the station. This was originally a skyrat addition because they don't know what balance in a video game is. Miners shouldn't get to play dota 2 and come back on station an hour into the game and be able to do whatever the fuck they want to whoever they want with zero recourse.

If you'd actually code the iteration i mentioned it wouldn't be good at all. Blood jaunt is coded with wizards in mind so of course it isn't balanced with miners having it , but why just remove it when it does fit the bubblegum theme and when it can be rewritten to be actually balanced

**Review by StrangeWeirdKitten — APPROVED** [review]

**Comment by StrangeWeirdKitten** [comment]
I like blood crawl specifically for kinky ERP. But that's my own personal preference for it. While I credit skyrat for making a better alternative to the mayhem bottle, this thing needs to go. Someone is free to rework it if they have a good idea on how to do such

**Pull request merged**